Ve.Metrics.StatsDClient.Tests/CastleWindsor/StatsDCountingInterceptorShould.cs

```diff
@@ -37,6 +37,13 @@ public void It_should_count_the_targeted_generic_method_and_log_to_statsd()
         }
 
         [Test]
+        public void It_should_count_the_targeted_nested_generic_method_and_log_to_statsd()
+        {
+            Service.TrackedGenericMethod<IEnumerable<object>>();
+            StatsdMock.Verify(x => x.LogCount("dependencies.ienumerable`1-object", It.IsAny<int>(), It.IsAny<Dictionary<string, string>>()), Times.Once);

```

Comment:
They are added by .NET when you get the type name and are removed by statsD when submitting.